lasagne/layers/pool.py

```diff
@@ -14,6 +14,39 @@
 ]
 
 
+def pool_output_length(input_length, pool_length, ceil=False):
+    '''Compute the output length of a pooling operator
+    along a particular dimension.
+
+    Parameters
+    ----------
+    input_length
+    pool_length
+        Shape of the input and pooling operator in the chosen dimension
+
+    ceil : bool
+        If True, size is rounded up.
+        If False (default), size is rounded down.
+
+    Returns
+    -------
+    output_length
+        * None if either input is None
+        * `input_length / pool_length` otherwise
+    '''
+
+    if input_length is None or pool_length is None:
+        return None
+
+    if ceil:
+        rounder = np.ceil
+    else:
+        rounder = np.floor
+
+    # Should this just be "input_length // pool_length" ?
+    return int(rounder(float(input_length) / pool_length))
```

Comment:
What about:
```python
if input_length is None or pool_length is None:
    return None
if ignore_border:
    # round down
    return input_length // pool_length
else:
    # round up
    return (input_length - 1) // pool_length + 1
```
(At least I'd suggest to change `ceil=False` to `ignore_border=False`, with inverted meaning, to match the `MaxPool2DLayer` constructor. I'm probably picky here, but `ceil` refers to the implementation of the shape computation, not to the operation performed by a pooling layer.)